simuleval/data/dataloader/dataloader.py

```diff
@@ -125,8 +125,8 @@ class IterableDataloader:
 
     @abstractmethod
     def __iter__(self):
-        ...
+        pass
```

Comment:
Changed to pass, since the 3.7 and 3.8 Python's black formatters have different versions and they treat this ellipsis differently. The keyword `pass` also implies coming back to this code later on.